library/src/morpheus-master/sass/modules/connection-manager/_base.scss

```diff
@@ -26,11 +26,14 @@
                     padding-right: 12px !important;
                 }
             }
-            &.current > a {
-                background-color: #FFFFFF !important;
-                color: $tool-tab-active-highlight-color !important;
-                font-weight: 600;
-                pointer-events: none;
+            &.current {
+                padding: 0;
+                a {
+                    background-color: #FFFFFF !important;
+                    color: $tool-tab-active-highlight-color !important;
+                    font-weight: 570;
```

Comment:
I'm just about to get rid of it, actually. The font weight still makes the tab jump about. You're right about the increments though. 570 will become 600. I thought it was more sophisticated and vectory than that. I should RTFM :)